modules/bibupload/lib/bibupload.py

```diff
@@ -23,7 +23,7 @@
 tables according to options.
 """
 
-__revision__ = "$Id$"
+__revision__ = "6a4b1d3"
```

Comment:
note this is just replacing the old dysfunctional `CVS` magic string replacement which hasn't worked since the migration to `git` with the latest commit hash -- that's better than before, but not optimal. however going to semver seems like overkill here